README.md

```diff
@@ -115,6 +115,8 @@
 - [stylelint-value-no-exposed-custom-properties](https://github.com/denisraslov/stylelint-value-no-exposed-custom-properties) - Disallow usage of exposed custom properties.
 - [stylelint-value-no-unknown-custom-properties](https://github.com/csstools/stylelint-value-no-unknown-custom-properties) - Disallow unknown custom properties.
 - [stylelint-z-index-value-constraint](https://github.com/kristerkari/stylelint-z-index-value-constraint) - Specify minimum and maximum constraint value for z-index.
+- [stylelint-stylistic](https://github.com/elirasza/stylelint-stylistic) - Enforce several stylistic rules.
```

Comment:
Thanks! Can you sort the plugins alphabetically?

Reply:
Done!